dist/RoktWSDK-Kit.common.js

```diff
@@ -0,0 +1,829 @@
+'use strict';
```

Comment:
Please remove everything from `/dist` except for the `.keep` file. We'll build the bundle as part of the release.